pkg/operator/workload/workload_openshiftapiserver_v311_00_sync.go

```diff
@@ -49,7 +49,7 @@ type nodeCountFunc func(nodeSelector map[string]string) (*int32, error)
 
 // ensureAtMostOnePodPerNode a function that updates the deployment spec to prevent more than
 // one pod of a given replicaset from landing on a node.
-type ensureAtMostOnePodPerNodeFunc func(spec *appsv1.DeploymentSpec) error
+type ensureAtMostOnePodPerNodeFunc func(spec *appsv1.DeploymentSpec, component string) error
```

Comment:
it looks like it needs wiring, this function is not being used. Compare with CAO

Reply:
looks like it is already wired, it just didn't have the right signature
```
	openShiftAPIServerWorkload := operatorworkload.NewOpenShiftAPIServerWorkload(
		operatorClient,
		operatorConfigClient.OperatorV1(),
		configClient.ConfigV1(),
		workloadcontroller.CountNodesFuncWrapper(kubeInformersForNamespaces.InformersFor("").Core().V1().Nodes().Lister()),
		workloadcontroller.EnsureAtMostOnePodPerNode,
		"openshift-apiserver",
		os.Getenv("IMAGE"),
		os.Getenv("OPERATOR_IMAGE"),
		kubeClient,
		controllerConfig.EventRecorder,
		versionRecorder)
```